lib/updaters/index.js

```diff
@@ -52,7 +58,13 @@ module.exports.resolveUpdaterObjectFromArgument = function (arg) {
    * We weren't able to resolve an updater for the argument.
    */
   if (!updater.updater) {
-    return false
+    /* Previously we just returned 'false' for this function, but that causes
+    an error immediately afterwards as we try to read the result as an object
+    with a field 'filename'. So better to bail here, it means something is
+    wrong, most likely someone has goofed making an updater. */
+    throw Error(
+      `Failed to create an updater for ${JSON.stringify(arg)}\n Please specifcy the updater \`type\` or use a custom \`updater\`.`
```

Comment:
Seems like there is a typo. 
Let's change `specifcy` to `specify`.

#typo-error

Reply:
Fixed, thanks @akashchandwani!